client/paddleflow/cli/log.py

```diff
@@ -64,19 +66,26 @@ def show(ctx, runid, jobid=None, pagesize=None, pageno=None, logfileposition=Non
 
 def _print_run_log(loginfo, out_format):
     """print run log """
-    headers = [
-        'run id', 'job id', 'task id', 'has next page', 'truncated', 'page no', 'page size', 'log content'
-    ]
     data = []
-    for item in loginfo:
+    run_loginfo = loginfo['runLog']
+    submit_loginfo = loginfo['submitLog']
+    data.append(["submit log:"])
+    data.append([submit_loginfo])
+    data.append([""])
+    data.append(["run log:"])
```

Comment:
It might be better to use list initialization: `data = [["submit log: "], ["submit_loginfo"], [""], ["runlog:"]]`